README.md

```diff
@@ -12,8 +12,8 @@ jobs:
   build:
     steps:
       - uses: actions/checkout@v2
-      - name: generate and commit diagrams
-        uses: abekoh/commit-plantuml-action@1.0.2
+      - name: PUML to PNG Commit & PR
+        uses: erzz/commit-plantuml-action@v1.0.0
```

Comment:
Please replace here with `abekoh/commit-plantuml-action@v1.1.0`. I'll tag 1.1.0 after I merge this PR.